include/mata/nfa.hh

```diff
@@ -537,18 +552,18 @@ struct Nfa
     //  dictionary in the attributes.
     std::unordered_map<std::string, void*> attributes{};
 
-    size_t max_state_;
+    size_t m_num_of_states;
```

Comment:
The prefix `m_` means member? Is it a standard. I'm used to use `_` at the end of a name of variable to denote private data member. But maybe things have changed.

Reply:
I am used to `m_xyz` denoting a member. I have seen this used more than `xyz_`, for example. It is just a matter of taste in my opinion, though.

What I like about `m_xyz` is that you cannot accidentally access the member variable unintentionally. It is much easier to miss the trailing underscore with autocomplete than forgetting that you have manually written `m_` at the beginning of the identifier.

In the library, we do not denote private data members in general. I had to use `m_` here only because there was a name clash between this member variable and the accessor method. Otherwise, it could be left without these special annotations.

Reply:
I have no problems with `m_`. I used `xyz_` elsewhere in library, but I think that it is a matter of taste.